snowflake/manifest.json

```
@@ -21,7 +21,7 @@
     "cost management"
   ],
   "type": "check",
-  "is_public": false,
+  "is_public": true,
```

lets not make snowflake public yet
```suggestion
  "is_public": false,
```